src/languages/types.ts

```diff
@@ -297,6 +297,8 @@ type HeldRequestParams = {comment: string};
 
 type DistanceRateOperationsParams = {count: number};
 
+type ReimbursementRateUnit = {unit: string};
```

Comment:
Can we use the `Unit` type from Policy instead? Also why is this an object?

Reply:
I typed the whole params object like in the other translations. I will rename it to ReimbursementRateParams

Reply:
Looking good now, thanks.